src/contents/encryption.rs

```diff
@@ -1,176 +1,125 @@
-use ursa::{
-    encryption::symm::prelude::*,
-    hash::{blake2::Blake2, Digest},
-    kex::KeyExchangeScheme,
-    keys::{PrivateKey, PublicKey},
+use crypto_box::{
+    PublicKey, 
+    Box,
```

Comment:
the crypto_box crate has now been released at v0.5 with support for XChaCha20Poly1305 :laughing:, we can actually now use chacha instead of salsa

Reply:
Replaced with ChaChaBox and ChaCha20poly1305 test vectors + test